stack/Stack.java

```diff
@@ -8,76 +8,39 @@
 import java.util.NoSuchElementException;
 
 class Stack<T> {
-    private int size;
     private LinkedList<T> stack;
-        
+
     public Stack() {   // Constructor to create empty Stack
-        stack = new LinkedList<> ();
-        size = 0;
+        stack = new LinkedList<>();
     }
 
     public void push(T data) { // Add element to Top of Stack
         stack.addLast(data);
-        size++;
     }
-        
-    public T pop() {    // Remove element from top of Stack
-        if (size == 0) {
-            throw new NoSuchElementException("Cannot Pop, Stack is empty");
-        }
-        size--;
+
+    public T pop() throws NoSuchElementException {    // Remove element from top of Stack
         return stack.removeLast();
     }
-       
-    public T peek() {   // Get the element at the Top of Stack
-        if (size == 0) {
-            throw new NoSuchElementException("No element left to Peek");
-        }
-        size--;
-        return stack.getLast();
-    }
-        
-    public int getSize() {  // Get size of Stack
-        return size;
-    }
-        
-    public void displayStack() {    // Display Stack
-        if (size == 0) {
-            throw new NoSuchElementException("No element in Stack to Display");
-        }
-        for (int i = size-1; i >= 0; i--) {
-                System.out.print(stack.get(i) + " ");
-        }
-        System.out.println();
-    }    
 }
 
 class StackTest {    // Tester Class to check the working of Stack
```

Comment:
The filename and public class containing the `main` should be consistent. Please modify accordingly.

I suggest that you add the main method to `Stack` class and self-instantiate there.